src/clojure_sandbox/4clojure.clj

```diff
@@ -0,0 +1,176 @@
+(ns clojure-sandbox.4clojure
+  (:require [clojure.string]))
+
+;; 21. Write a function which returns the Nth element from a sequence.
+(defn nth-element [[one & others :as sequence] position]
```

Comment:
I understand the struggle with trying to avoid shadowing first and finding a good name for this. But, one is confusing.

Perhaps...

```clojure
[current & rest]
```

Reply:
I've renamed them to `[current & remaining]` because `rest` shadows the `rest` function.